### PR TITLE
Affiliates

### PR DESCRIPTION
**Pull request by nathantrz**
This pull request adds an affiliates section.

### CODE REVIEW

**Comment by nathantrz** [comment]
@jonmspencer we can probably make this section work for mobile if you want. Let me know what you want to do. Other than mobile, it is ready for review.

**Comment by jonmspencer** [comment]
@nathantrz overall looking good. There is far too much padding on this section. Can you optimize? 

screen shot 2016-08-29 at 1 08 26 pm

**Comment by nathantrz** [comment]
@jonmspencer Padding optimized. 😺

**Comment by jonmspencer** [comment]
@nathantrz the final order of logos should be as follows: 

Frontier, AirBerlin, Aegean Airlines, and Copa.

File: wp-content/themes/cloudnine/assets/styles/modules/_affiliates.scss [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    float: left;
+    background-position: center center;
+    background-size: contain;
+    background-repeat: no-repeat;
+    content: "";
+    height: 3em;
+    margin: 0 4.1666%;
+    width: 15%;
+
+    &:first-of-type, &:last-of-type {
```
Comment (jonmspencer) [inline wp-content/themes/cloudnine/assets/styles/modules/_affiliates.scss]: @nathantrz You can remove the first and last of type entirely here and keep even margins on all logos.

File: wp-content/themes/cloudnine/assets/styles/modules/_affiliates.scss [hunk 2]
```diff
@@ -0,0 +1,41 @@
+.affiliates {
+  padding: 3.5em 0;
+
+  h3 {
+    margin-bottom: $double-minor-space;
+  }
+
+  .affiliate {
+    @include better-box;
+    float: left;
```
Comment (jonmspencer) [inline wp-content/themes/cloudnine/assets/styles/modules/_affiliates.scss]: @nathantrz you can also remove this float and let the elements naturally be centered due to the parent's centered text alignment. That way if there is a variable number they stack from the center and not the left.

**Comment by nathantrz** [comment]
@jonmspencer I have reordered the logos based on the above note. I also removed the float left.

**Comment by jonmspencer** [comment]
@nathantrz there seems to be a fifth div left in the structure? It's throwing off the spacing. Also, the frontier image looks a little rough. Mind re-doing that?

**Comment by nathantrz** [comment]
@jonmspencer I removed the errant affiliate div, and I replaced the frontier logo image in assets/images.

**Comment by jonmspencer** [comment]
@nathantrz I decided to replace them all with image svgs, but looking good! Merging.

**Pull request merged**